Readme.md

```diff
@@ -1477,3 +1477,53 @@ You can define any valid callable (static class method, closure, etc.) for it:
 ```
 
 The parameters received are your regular "resolve" function signature.
+
+### Migrating from https://github.com/folkloreinc/laravel-graphql
+Formerly also known as https://github.com/Folkloreatelier/laravel-graphql
+
+Both codebases are very similar and, depending on your level of customization, the migration may be very quick.
+
+Note: this migration is written with version 2.* of this library in mind.
+
+The following is not a bullet-proof list but should serve as a guide. It's not an error if you don't need to perform certain steps.
+
+**Make a backup before proceeding!**
+
+- `composer remove folklore/graphql`
+- if you've a custom ServiceProvider or did include it manually, remove it. The point is that the existing GraphQL code should not be triggered to run.
+- `composer require rebing/graphql-laravel`
+- Publish `config/graphql.php` and adapt it (prefix, middleware, schemas, types, mutations, queries, security settings, graphiql)
+- Change class references:
+  - from `Folklore\`
+  - to `Rebing\`
+- The signature of the method fields changed:
+  - from `public function fields()`
+  - to `public function fields(): array`
+- The signature of the method getFields changed:
+  - from `public function getFields()`
+  - to `public function getFields(): array`
+- The signature of the method interfaces changed:
+  - from `public function interfaces()`
+  - to `public function interfaces(): array`
+- The signature of the method types changed:
+  - from `public function types()`
+  - to `public function types(): array`
+- The signature of the method type changed:
+  - from `public function type()`
+  - to `public function type(): \GraphQL\Type\Definition\Type`
+- The signature of the method args changed:
+  - from `public function args()`
+  - to `public function args(): array`
+- The signature of the method queryContext changed:
+  - from `protected function queryContext($query, $variables, $schema)`
+  - to `protected function queryContext()`
+- The trait `ShouldValidate` does not exist anymore; the provided features are baked into `Field`
+- The first argument to the resolve method can be `null` know (instead of an empty array)
+- If you're using custom Scalar types:
+  - the interface `Folklore\GraphQL\Support\Contracts\TypeConvertible` is gone and there's no replacement (needed)
+  - the signature of the method parseLiteral changed:
+    - from `public function parseLiteral($ast)`
+    - to `public function parseLiteral($valueNode, ?array $variables = null)`
+  - Some node types changed, just one example
+    - from `GraphQL\Language\AST\StringValue`
+    - to `GraphQL\Language\AST\StringValueNode`
```

Comment:
Going to remove this part, as this is actually unrelated and specific to the `graphql-php` library